### ISSUE TITLE
Alert of PostgresqlConfigurationChanged cannot simply work

### ISSUE THREAD

**Issue by zhngcho** [samber/awesome-prometheus-alerts#256]
### how it happen
- navigate to https://awesome-prometheus-alerts.grep.to/rules
- goto 2.2.23. Postgresql configuration changed
- click [copy]
- paste in my rules file
- promtool check config prometheus.yml

### why it happen
- There are some [ and ] characters in the alert rule expr above, which are yml meta character 

### how to solve
- Add  single quote (' ) character around the expr

### reason for help
- I am really not good at html things

**Comment by paulfantom** [comment]
I wonder why would someone even alert on that in the first place?! What's the action item for such an alert especially considering this had to be triggered by someone.

---

@samber I am constantly watching this repo as it has a ton of github stars and good SEO in google and sometimes I wonder why is it spreading such bad practices? Especially when it comes to alerts like the one mentioned in this issue which clearly are not aligned with [prometheus best practices](https://prometheus.io/docs/practices/alerting/).

**Comment by ccppprogrammer** [comment]
@bretton After applying your patch, `prometheus` started throwing an error
```
Feb 16 19:51:18 prometheus[26321]: ts=2022-02-16T19:51:18.083Z caller=manager.go:609 level=warn component="rule manager" group=AllInstances msg="Evaluating rule failed" rule="alert: PostgresqlConfigurationChanged\nexpr: '{__name__=~\"pg_settings_.*\"} != on(__name__) {__name__=~\"pg_settings_([^t]|t[^r]|tr[^a]|tra[^n]|tran[^s]|trans[^a]|transa[^c]|transac[^t]|transact[^i]|transacti[^o]|transactio[^n]|transaction[^_]|transaction_[^r]|transaction_r[^e]|transaction_re[^a]|transaction_rea[^d]|transaction_read[^_]|transaction_read_[^o]|transaction_read_o[^n]|transaction_read_on[^l]|transaction_read_onl[^y]).*\"}\n  offset 5m'\nlabels:\n  severity: warning\nannotations:\n  description: |-\n    Postgres Database configuration change has occurred\n      VALUE = {{ $value }}\n      LABELS = {{ $labels }}\n  summary: Postgresql configuration changed (instance {{ $labels.instance }})\n" err="found duplicate series for the match group {__name__=\"pg_settings_allow_system_table_mods\"} on the right hand-side of the operation: [{__name__=\"pg_settings_allow_system_table_mods\", instance=\"xxx1:9187\", job=\"postgres_exporter\", server=\"localhost:5432\"}, {__name__=\"pg_settings_allow_system_table_mods\", instance=\"xxx2:9187\", job=\"postgres_exporter\", server=\"localhost:5432\"}];many-to-many matching not allowed: matching labels must be unique on one side"
```

**Comment by bretton** [comment]
> @bretton After applying your patch, `prometheus` started throwing an error

Sorry about that, it should all be on one line, surrounded by single quotes. 

However it's been removed from our usage in a later commit b2753c61626548744540ba66353a2771356db4bd

**Comment by samber** [comment]
Thanks for the report.

I made a fix for the rule: #281

I changed severity to `info`.
